fdbserver/TagPartitionedLogSystem.actor.cpp

```diff
@@ -167,7 +167,7 @@ struct TagPartitionedLogSystem : ILogSystem, ReferenceCounted<TagPartitionedLogS
 	UID recruitmentID;
 	int repopulateRegionAntiQuorum;
 	bool stopped;
-	std::set<int8_t> pseudoLocalities;
+	std::set<int8_t> pseudoLocalities; // Q: What is pseudo-locaility?
```

Comment:
pseudo-localities represent special localities that will be mapped to tagLocalityLogRouter. Right now the code base only has tagLocalityLogRouterMapped and I'll add another tagLocalityBackup.

tagLocalityLogRouterMapped is used when LogRouters pop tag from primary TLogs. The TLog receives tagLocalityLogRouterMapped, tracks how much pseudo localities have popped, and then pops tagLocalityLogRouter versions up to min(all pseudo localities).